src/debugger.ts

```diff
@@ -173,7 +175,6 @@ export class Debugger
         "rdbg",
         "--open",
         "--command",
-        `--sock-path=${sockPath}`,
```

Comment:
This is still necessary for launch. In launch operations, the extension defines where the socket should be placed, so that it knows how to auto-connect.

We only want to change attach.

Reply:
The point is that we maybe don't want to hardcode those sockets and want to let rdbg decide. We then extract them from debugger output. The implementation provided handles both ends, see your other comment. 

> Yeah, this is not necessary if we keep our standard socket path for launch.

Reply:
@st0012 I think this is fine, but wanted to double-check to see if you have any concerns.